### ISSUE TITLE
When Master is 100: TypeError: values is null

### ISSUE THREAD

**Issue by ktonekaboni** [vaurelios/Alsa-Mixer#1]
Hi,

I found an error in the extension in line 65:

```
_getVolume: function() {
    let cmd = GLib.spawn_command_line_sync(
            'env LANG=C /usr/bin/amixer get %s'.format(MIXER_ELEMENT));
    let re = /\[(\d{1,2})\%\]/m;
    let values = re.exec(cmd[1]);
    return values[1];
},
```

The regexp doesn't match a volume of 100%

```
user@host:~$ env LANG=C amixer get Master
Simple mixer control 'Master',0
  Capabilities: pvolume pswitch penum
  Playback channels: Front Left - Front Right
  Limits: Playback 0 - 31
  Mono:
  Front Left: Playback 31 [100%] [0.00dB] [on]
  Front Right: Playback 31 [100%] [0.00dB] [on]
```

Unfortunately my js skills are approx 0 and I couldn fix it by myself to send you a pull request.

Bes

Keywan

**Comment by ktonekaboni** [comment]
I forgot to mention, that I'm using Gnome 3.4.2 on Debian Wheezy

**Comment by guillaume-uH57J9** [comment]
Me too. Master volume is at 100% and the extension won't start.

I've used my mad JS skills to make a patch. It beats my own record of the tinyest patch: 1 character change. Pull request on its way.

**Comment by ktonekaboni** [comment]
Thanks. I did this change before, too. Off course 3 instead of to 2 matches 100%. But the code is more broken, the slider reacsts a bit crazy. What are the values for the slider to be right and left (from 0 to 100 or different)?

**Comment by guillaume-uH57J9** [comment]
Well 3 instead of 2 fixes the problem described in this issue. If you notice other issues you should probably log other bugs.

**Comment by ktonekaboni** [comment]
Ooops. Sorry, the solution is found, but not yet fixed in the code.

**Comment by guillaume-uH57J9** [comment]
Is there an issue with my pull request, or is the maintainer simply otherwise busy?

**Comment by texasflood** [comment]
I have fixed both these issues in this fork: https://github.com/texasflood/Alsa-Mixer

**Comment by ktonekaboni** [comment]
@texasflood great. do you plan to commit your fork to extensions.gnome.org?

**Comment by texasflood** [comment]
Yes, but once I have used the extension for longer and ensured there are no more bugs, I've only been using it for three days. I'll also clean it up a bit and see if I can add any features, some things about it still annoy me. Any requests? 

For now, you could just replace the extension.js file with my version on your computer and you will get the fixes

**Comment by guillaume-uH57J9** [comment]
Thanks for the fork.
Great initiative; this repository seems unmaintained
